Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -765,6 +765,16 @@ define([
             u_waterMaskTranslationAndScale : function() {
                 return this.waterMaskTranslationAndScale;
             },
+            u_minimumHeight : function() {
```

Comment:
Ah, `u_scaleAndBias` is a matrix.  Can we combine the min and max height into a `vec2`?